src/Sulu/Bundle/ContactBundle/DataFixtures/ORM/LoadDefaultTypes.php

```diff
@@ -94,6 +98,7 @@ public function load(ObjectManager $manager)
 
         // fax types
         $metadata = $manager->getClassMetaData(FaxType::class);
+        $metadata->setIdGenerator(new \Doctrine\ORM\Id\AssignedGenerator());
```

Comment:
With the (existing) modification of the metadata it was ensured that the types always have the same id, so that they code be referenced in the code by hardcoding the id. This approach only worked in Mysql but not on Postgres. In Postgres a defined IdGenerator is necessary.